### PR TITLE
add custom per column align

### PR DESCRIPTION
**Pull request by pkit**
adds `SetColumnAlignment` to `Table` interface
solves #43

### CODE REVIEW

File: table.go [hunk 1]
```diff
@@ -210,6 +213,22 @@ func (t *Table) SetAlignment(align int) {
 	t.align = align
 }
 
+func (t *Table) SetColumnAlignment(keys []int) {
```
Comment (tamird) [inline table.go]: doesn't seem safe to call this multiple times. is that intended?

Comment (pkit) [reply table.go]: That's the behavior of all other `Set*` functions in this pkg.
I suspect that's because things are usually added column-by-column.

**Comment by pkit** [comment]
@tamird bump. Is there something else that should be done?
I can obviously refactor the code and make it safe, but it would mean that I need to change behavior of at least `SetHeader()` and `SetFooter()`

**Comment by tamird** [comment]
I'm not a collaborator on this repo.

**Comment by pkit** [comment]
Oh, sorry.
You had a "Commiter" tag. My bad.

**Comment by pkit** [comment]
@mattn please review the PR. お願いします。

**Comment by mattn** [comment]
Thanks for your contribution. I'll look into it tonight.

**Comment by knz** [comment]
Also curious about this feature!

**Comment by mattn** [comment]
Thank you. We should add the check that verify several slices in future.

**Pull request merged**